R/display_scatter.r

```diff
@@ -45,87 +44,106 @@
 #' `h`
 #' @param edges A two column numeric matrix giving indices of ends of lines.
 #' @param paused whether the widget should be initialised in the 'paused' state
+#' @param scale_factor used as a multiplier for the point coordinates so they
+#' are displayed on a sensible range. Defaults to the reciprocal of maximum distance
+#' from a point to the origin.
 #' @importFrom rlang `%||%`
 #' @export
 #' @examples
-#' animate_tour(tourr::flea, tourr::grand_tour(3), display_scatter())
-display_scatter <- function(mapping = NULL,
+#' detour(tourr::flea, tour_aes(projection = -species, colour = species)) %>%
+#'   tour_path(grand_tour(3), fps = 60) %>%
+#'   display_scatter(alpha = 0.7, axes = FALSE)
+#' @export
+display_scatter <- function(d,
```

Comment:
```suggestion
display_scatter <- function(x,
```

Conventionally, use `x`.

Reply:
I believe the convention is to use `p` for plots, which begs the question: would we want the `detour` object to be plottable from the start? I.e. have `detour` return an `htmlwidget` with a default tour path and display method?

Reply:
Either `x` or `p` works, I suppose. I'd like to have `tour_path()` to be traceable before plotting. `display_*()` makes the `detour` object plottable.